### ISSUE TITLE
Visual appeal of week view in What's On

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#490]
hernehill.org standarduser

Looking at the week view on what's on I was wondering if there was anything that could be done to get it to look a little nicer? It looks like a non-designer has had a go at it, something that I would come upo with. It works, you can see stuff but it doesn't look as slick as the rest of the site.
How much of this module is accessible for tweaking? Colours? layout/ font?

![image](https://cloud.githubusercontent.com/assets/6060985/6306313/55da2928-b92a-11e4-8abd-61b4075d924e.png)

**Comment by djwesto** [comment]
Hi Giles.

You've only just noticed this? I mentioned the other day that I'd love to get stuck into the typography of this as well as the deeper pages once you get inside the forum. Have you checked those pages yet either?

I'm afraid that if you want to launch in a week or so, there's almost certainly no way I'll be able to style these all up correctly in time, as well as write all your documentation.

I will sit with @pmackay some time next week and take a look at what we can do, but I'm afraid this is probably a v1.1 issue.

I don't like the way it looks at all, but it works and should be OK for launch.

**Comment by GilesGibson** [comment]
No, not only just noticed it but wanted to add to the list of desirables and issues to tackle in the future - hence on the wishlist.
